src/AbstractFirestoreRepository.ts

```diff
@@ -108,14 +108,15 @@ export abstract class AbstractFirestoreRepository<T extends IEntity>
     if (!doc.exists) {
       return null;
     }
+    
     // tslint:disable-next-line:no-unnecessary-type-assertion
     const entity = plainToClass(
-      this.colMetadata.entity as any,
+      this.colMetadata.entity,
```

Comment:
Interested in why suddendly this `any` is not required, I was getting compiler errors before 🤔